tools/cmake/xcode.toolchain.cmake

```diff
@@ -19,7 +19,6 @@ set(CMAKE_XCODE_ATTRIBUTE_MACOSX_DEPLOYMENT_TARGET_CATALYST_NO "10.9")
 set(CMAKE_XCODE_ATTRIBUTE_MACOSX_DEPLOYMENT_TARGET_CATALYST_YES "10.15")
 
 set(CMAKE_XCODE_ATTRIBUTE_ARCHS "$(ARCHS_STANDARD)")
-set(CMAKE_XCODE_ATTRIBUTE_ARCHS[sdk=iphoneos*] "armv7 arm64")
```

Comment:
This appears to be unnecessary now and both Xcode 13 and 14 do the right thing with it unset (13 builds armv7 and arm64, 14 builds just arm64 due to no longer supporting armv7).